go.mod

```diff
@@ -10,17 +10,17 @@ require (
 	github.com/moby/locker v1.0.1
 	github.com/moby/sys/mountinfo v0.6.2
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/opencontainers/image-spec v1.1.0-rc2.0.20221005185240-3a7f492d3f1b
+	github.com/opencontainers/image-spec v1.1.0-rc5
```

Comment:
image-spec has been release, use v.1.1.0 instead of rc? https://pkg.go.dev/github.com/opencontainers/image-spec@v1.1.0/specs-go/v1

Reply:
Thanks for the heads up, I'll have a look at the differences between the 2 versions.